arch/arm/core/aarch64/vector_table.h

```diff
@@ -27,7 +27,7 @@
 #include <linker/sections.h>
 
 GTEXT(__start)
-GTEXT(_vector_table)
+GDATA(_vector_table)
```

Comment:
`_vector_table`  symbol points to executable code `TEXT` 
I am not too sure how this impacts.
SECTION_SUBSEC_FUNC(exc_vector_table,_vector_table_section,_vector_table)

Reply:
@sandeepbrcm this is only to instruct linker to export the global symbol as either %function or %object.
AFAICS, in AARC64 the _vector_table symbol is not referenced, either.

Reply:
`_vector_table` contains code to be executed on exceptions, that's why `%function` is more suitable. What would be impact of using `%object`? If no impact, i.e. the vector table still remains in executable code `TEXT` segement, then okay to use `GDATA`.
@ioannisg `_vector_table` symbol is referenced here:
https://github.com/zephyrproject-rtos/zephyr/blob/master/arch/arm/core/aarch64/reset.S#L57

Reply:
Agreed @Abhishek-brcm , but VBAR is something like VTOR in Cortex-M, right? So some LSBs are masked, anyways. We have the same in Cortex-M (VTOR gets assigned the address of the vector table, but not through the `_vector_table` symbol, but via an auxiliary linker symbol called `vector_start` which does not have any Thumb flag set :) )